docs/contributing/marketplace.md

```diff
@@ -7,31 +7,34 @@ Cortex XSOAR v6.0 introduces a Marketplace which is the central location for ins
+2. After you contribute the content entity a message displays explaining that your contribution is ready for review, and includes a link to a form you need to fill in to complete your contribution. You will receive an email with a link to the form.
 
 3. Completing the form
 
-   1. **Create a New Pack**  
+   ### **Contribute a New Pack**  
 
-          These instructions are only intended for creating a new pack. If you have a pack that you need to update, follow the instructions in the _Update an Existing Pack_ section.
-
-          1. Fill in additional details, such as, pack name, author, and description.
-          2. Log in to your GitHub account so you can participate in the review process of the pull request that will be opened for your content pack.
-          3. Sign the [Palo Alto Networks Contributor License Agreement](https://github.com/demisto/content/blob/master/docs/cla.pdf).
-
-   2. **Update an Existing Pack**
-
-          If your contribution is an update to an existing pack, complete the following steps. (See _[Notes](#notes)_ for more information regarding updating an existing pack.) 
-
-          1. Select _Update Existing Pack_ from the _Select Contribution Mode_ dropdown.
-          2. Select the pack that you wish to update from the _Select Existing Pack_ dropdown.
+      These instructions are only intended for creating a new pack. If you have a pack that you need to update, follow the instructions in the _Contribute to an Existing Pack_ section.
+      
+          1. Select "Create New Pack" from the "Select Contribution Mode" dropdown.
+          2. Fill in additional details, such as, pack name, author, and description.
           3. Log in to your GitHub account so you can participate in the review process of the pull request that will be opened for your content pack.
           4. Sign the [Palo Alto Networks Contributor License Agreement](https://github.com/demisto/content/blob/master/docs/cla.pdf).
 
+   ### **Contribute to an Existing Pack**
+   You can contribute new entities or update existing entities for an existing pack.
+   
+         1. Select "Update Existing Pack" from the "Select Contribution Mode" dropdown.
+         2. Select the pack that you wish to update from the "Select Existing Pack" dropdown.
+         3. Log in to your GitHub account so you can participate in the review process of the pull request that will be opened for your content pack.
+         4. Sign the [Palo Alto Networks Contributor License Agreement](https://github.com/demisto/content/blob/master/docs/cla.pdf).
+
+   
+	_Contribute to an existing content entity_: Duplicate the entity you want to contribute to. For example, if you want to update the `AbuseIPDB` integration, you need to duplicate it  and save the duplicate with your changes as `AbuseIPDB_copy`.
+	
 
 4. After you submit the form you will be redirected to a page that informs you that your pack was received and is being processed.  
 
```

Comment:
For number 6 I think you can remove this bit,
`(Each contributor can only modify files in content packs that they contributed).`

For number 7, I think language-wise it can be rewritten to be more accurate (some of this may no longer be true/accurate so verify that this is actually the behavior still today),
```
    A documentation template is automatically generated for the content entities included in your contribution.
    You will need to review the documentation (README.md) and modify it according to XSOAR standards.
    The files to be reviewed will be listed in a comment on the pull request.
    See the [Documentation article](https://xsoar.pan.dev/docs/documentation/readme_file) for more information.
```